CRM/Mailing/BAO/Mailing.php

```diff
@@ -1788,7 +1609,7 @@ public static function create(&$params, $ids = array()) {
 
     // Create parent job if not yet created.
     // Condition on the existence of a scheduled date.
-    if (!empty($params['scheduled_date']) && $params['scheduled_date'] != 'null' && empty($params['_skip_evil_bao_auto_schedule_'])) {
+    if (!empty($params['scheduled_date']) && $params['scheduled_date'] != 'null') {
```

Comment:
I have opened #11653 to e-instate this check - I assume is was removed on the assumption that it was not being used but it's actually accessible via the api. We should merge that to 4.7.31